apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -44,3 +44,73 @@
     line-height: 1.1875;
   }
 }
+
+.learningGoalRow {
+  border-bottom: 1px solid $neutral_dark40;
+  height: 40px;
+  :hover {
+    background: none;
+  }
+}
+
+.learningGoalHeader {
+  display: flex;
+  height: 40px;
+  padding: 0 16px;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+  line-height: 154%;
+}
+
+.learningGoalHeaderLeftSide {
+  display: flex;
+  align-items: center;
+  gap: 16px;
+  font-family: $gotham-bold;
+
+  span {
+    font-size: 14px;
+    font-weight: 350;
+  }
+
+  .arrowIcon {
+    font-weight: 900;
+    font-size: 16px;
+  }
+}
+
+.learningGoalHeaderRightSide {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 16px;
+  font-weight: 325;
+  line-height: 154%;
+}
+
+.aiToken {
+  display: flex;
+  height: 20px;
+  padding: 4px 8px;
+  justify-content: center;
+  align-items: center;
+  gap: 4px;
+
+  border-radius: 30px;
+  border: 1px solid $product_affirmative_default;
+  color: $product_affirmative_default;
+  background: $product_affirmative_100;
+
+  text-align: center;
+  font-feature-settings: 'clig' off, 'liga' off;
+  font-family: $gotham-extra-bold;
```

Comment:
@molly-moen shouldn't we use typography element here instead of custom text styles?

Reply:
Did you mean to tag Molly or me? Either way, there wasn't a Typography element called out in the figma so I used the custom styles. Do you have a recommendation for which element to use?

Reply:
Sorry, had to specify it - I did mean @molly-moen . 
Looked into Figma and I see that this text is custom text, not a typography component. (meaning @bethanyaconnor from FE side all is ok 👍 , purely design question)

Reply:
The text in the AI token. This one:

![Screenshot 2023-08-08 at 1 41 11 PM](https://github.com/code-dot-org/code-dot-org/assets/46464143/896a7019-8fd0-4359-bce4-bcb55d7b0dc8)

Reply:
(fwiw, I don't think this discussion blocks me)

Reply:
Ah, thank you. I think the formatting on this was an oversight on my part - I created the component before we had the full type styles defined. The equivalent in the new type styles will be Body/Body4/Bold.